bower.json

```diff
@@ -1,12 +1,14 @@
 {
   "name": "o-typography",
-  "main": ["main.scss"],
+  "main": [
+    "main.scss"
+  ],
   "ignore": [
     ".travis.yml"
   ],
   "dependencies": {
-    "o-colors": ">=3.4 <4",
-    "o-hoverable": ">=0.1.1 <4",
-    "o-fonts": "^2.0.0"
+    "o-colors": "^4.0.1",
+    "o-fonts": "^3.0.0",
+    "o-grid": "^4.3.1"
   }
```

Comment:
we are missing the fontface observer, it needs to be present in bower.json instead of package.json to ensure bower installations work correctly